webapps/knowrob/webrob/templates/knowrob.html

```diff
@@ -82,11 +82,14 @@
       var videoFile = libraryFile.replace(new RegExp('queries', 'g'), 'video');
       var wsproto = 'ws' + (location.protocol === 'https:'?'s':'');
       var rosUrl = wsproto + '://{{ host_url }}/ws/{{ container_name }}/';
+      var auth = '{{ authentication }}';
```

Comment:
Ok for other routes? Might be empty string if `authentication` not defined in python code.

Also sure that '' are needed around {{ authentication }} ? Because like this it's allways a string but it should be a boolean.